MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -18,6 +18,7 @@ internal protected enum SurfaceType
 
 		internal int width;
 		internal int height;
+        internal int ArraySize;

```

Comment:
I made it pascal-case because it's `internal`, not `private` - it is referenced from `RenderTargetBinding`. (And it's a field, not a property, for efficiency.)

Should I still rename it?